src/test/java/com/speaker/service/AccountServiceImplTest.java

```diff
@@ -88,4 +95,49 @@ public void accountNotCreatedWhenCityNameIsNull() {
         AccountDTO accountDTO = generateAccountDTO(generateCountryDTO(CountryName.UKRAINE, generateCityDTO(null)));
         assertThat(accountService.create(accountDTO), is(Response.FALSE));
     }
+
+    @Test
+    public void addFriendSuccess() {
+        FriendDTO friendDTO = generateFriendDTO("Ivan Mazur", "Victor Mazur");
+        Friend friend = generateFriend(1, 2);
+        when(accountRepository.findAccountIdByNameAndLastName("Ivan", "Mazur")).thenReturn(Optional.of(1));
```

Comment:
"Ivan", "Victor", "Mazur" repeated many times, I think we can extract this in constants and reuse it where needed, the same for ids

Reply:
made constants. Changes made

Reply:
please ids also extract in constants